### PR TITLE
integration: fix NewClientV3 error messages

### PR DESCRIPTION
**Pull request by gyuho**
/cc @xiang90

### CODE REVIEW

**Comment by xiang90** [comment]
I do not quite understand what this pr is trying to achieve. More explanation?

**Comment by gyuho** [comment]
@xiang90 Just fixing the wrong error message?

https://github.com/coreos/etcd/blob/master/integration/v3_grpc_test.go#L965-L973

It doesn't use TLS in this test, so we should just `t.Fatal(err)`?

**Comment by gyuho** [comment]
Oh wrong test. Will fix.

**Comment by gyuho** [comment]
Just fixed. Now it matches what I said. Sorry!

File: integration/v3_grpc_test.go [hunk 1]
```diff
@@ -990,7 +990,7 @@ func TestGRPCRequireLeader(t *testing.T) {
 
 	client, err := NewClientV3(clus.Members[0])
 	if err != nil {
-		t.Fatalf("expected tls client (%v)", err)
+		t.Fatal(err)
```
Comment (xiang90) [inline integration/v3_grpc_test.go]: t.Fatal("cannot create client: %v",err)

**Comment by xiang90** [comment]
also fix the title of pull request?

**Comment by gyuho** [comment]
@xiang90 Addressed. PTAL.

**Comment by xiang90** [comment]
lgtm

**Comment by gyuho** [comment]
Thanks. Merging in greens.

**Pull request merged**